logstage/logstage-config/src/main/resources/logstage-reference.conf

```diff
@@ -0,0 +1,46 @@
+// here you need to define all rendering policies you want to use
```

Comment:
Could you move the file to test scope, add a test for it, and write the docs in `/doc/paradox/logstage/config.md` and in scaladoc comment for `LogConfigModule`?